apps/test/unit/lib/levelbuilder/rubrics/RubricsContainerTest.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
-import {mount} from 'enzyme';
+import {mount, shallow} from 'enzyme';
+import {act} from 'react-dom/test-utils';
```

Comment:
hm I don't know what this is. Is it from React Testing Library? It's interesting that it works here 🤔